packages/notifier/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "files": [
     "src/",
+    "tools",
```

Comment:
Is this an innovation I should be evaluating or following an established precedent? I suppose I see precedent:

```
agoric-sdk/packages$ echo */tools
agoric-cli/tools SwingSet/tools zoe/tools
```